iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/wal/io/WALInputStream.java

```diff
@@ -226,15 +231,18 @@ private void loadNextSegmentV2() throws IOException {
         throw new IOException("Unexpected end of file");
       }
       compressedBuffer.flip();
-
       IUnCompressor unCompressor = IUnCompressor.getUnCompressor(segmentInfo.compressionType);
       unCompressor.uncompress(compressedBuffer, dataBuffer);
+      MmapUtil.clean(compressedBuffer);
```

Comment:
The direct buffer is not released if an exception occurred in previous code.

Reply:
It should be noted that clean here is only for early release of unwanted off-heap memory, but even if it is not freed, off-heap memory may eventually be freed by a Full GC.

With a non-memory leak like this, I feel like we don't need to add a lot of try finally to make the code less maintainable, what's your opinion?

Reply:
ok